### DISCUSSION TITLE
115挂载提示 failed get objs: failed to list objs: invalid character 'a' looking for beginning of value

### DISCUSSION THREAD

**Discussion by duojiaqi123** [AlistGo/alist#7190] [Q&A] [answered]
115挂载提示 failed get objs: failed to list objs: invalid character 'a' looking for beginning of value
![屏幕截图 2024-09-16 213527](https://github.com/user-attachments/assets/8146d0b2-47ef-4cf0-8200-d882e265bc01)

**Comment by sun3u** [comment]
+1，下午好好的，晚上就不行了，更新了alist，重新添加115都不行

**Comment by leooooooooo** [comment]
有没有大佬说一下，是115下午挂了还没完全修复好的问题，还是3.37版本的问题呢

**Comment by RockyZhous** [comment]
应该不是3.37版本的问题，我这边有个旧版本的还在win7上可运行的，也是报同样的错误。

**Reply by huiyy-cn** [reply]
> 应该不是3.37版本的问题，我这边有个旧版本的还在win7上可运行的，也是报同样的错误。

115的问题，alist几个版本都这个情况

**Comment by alhm008** [comment]
现在仍然是提示这个错误，无法使用。难道是115在下午的修复中，更新了什么端口之类的？

**Reply by alhm008** [reply]
刚刚看到有网友反映CloudDrive2也无法挂载115网盘了，提示＂Error：Status(StatusCode="PermissionDenied"
Detail="deserialize error InvalidJsonString("access deny") content:
access deny" ，看来115在下午的紧急修复中，肯定是针对WebDAV挂载端口作了调整修改，现在也只能希望alist的大神能尽快修复115网盘新出现的挂载问题了！

**Reply by shanhai2333** [reply]
封禁了桌面的ua，之前115就下架了桌面，这次顺便就禁用了吧，alist写死的ua，cd2已经可以改ua挂上了

**Comment by zxkke** [comment]
一样的问题

**Comment by alhm008** [comment]
xhofe大神深夜不辞辛劳，已神速解决，目前各客户端均也更新，亲测完美！万分感谢xhofe大神！

**Reply by leooooooooo** [reply]
感谢大神，更新了3.37.3下载正常了，但上传依旧失败，不知道是不是我自己的问题

**Reply by hzt2000** [reply]
> xhofe大神深夜不辞辛劳，已神速解决，目前各客户端均也更新，亲测完美！万分感谢xhofe大神！

谢谢，有无具体攻略？Alist的说明没有Xhofe大神的

**Reply by alhm008** [reply]
在alist官网上去选择最新版本号为3.37.3的更新就行。

**Reply by hzt2000** [reply]
> 在alist官网上去选择最新版本号为3.37.3的更新就行。

是的，成功了，谢谢

**Reply by qinsijie** [reply]
> 感谢大神，更新了3.37.3下载正常了，但上传依旧失败，不知道是不是我自己的问题

我也是下载正常了，但是不能上传，你的问题解决了吗？

**Reply by ttttC9** [reply]
Android版资源下载下来怎么用呢，我看解压出来都不是apk文件，四个都下载了，谢谢大哥

**Reply by fen9wu** [reply]
> Android版资源下载下来怎么用呢，我看解压出来都不是apk文件，四个都下载了，谢谢大哥

有个alistlite专门给安卓开发的，试试那个

**Reply by hzt2000** [reply]
> > 感谢大神，更新了3.37.3下载正常了，但上传依旧失败，不知道是不是我自己的问题
> 
> 我也是下载正常了，但是不能上传，你的问题解决了吗？

确实115好像对来自Alist的上传限制了？只有文件夹可以上传，具体文件都上传不成功。官方115app上传是正常没问题的

**Comment by baymax222** [comment]
安卓版的用模拟机doc的还是不行，termux版的

**Comment by dajie111** [comment]
更新到最新版115恢复了

**Reply by applemomoka** [reply]
更新到最新版115是啥意思，不是alist的问题么，更新115有啥用呢

**Reply by qinsijie** [reply]
> 更新到最新版115是啥意思，不是alist的问题么，更新115有啥用呢

是更新alist到最新版

**Comment by fengzhou** [comment]
app store 上啥时候可以更新啊。

**Reply by 77095669** [reply]
希望开发者大神快更新吧  我一天看八回applestore
两天都只能看直播了

**Comment by qinsijie** [comment]
更新alist到3.7.3，能挂载115，能下载文件，但是不能上传，是怎么回事？

**Comment by hnzhaihui** [comment]
本人的安装的iStoreOS 22.03.6 2024032917 ，商店里的alist还是3.33.0，无法升级3.37.3，要是alist能在web页里面升级最新版那真是太棒了，希望xhofe大神能在alist里加入这个功能就好了！

**Reply by fen9wu** [reply]
手动更新啊，不用等商店的版本

**Reply by hnzhaihui** [reply]
> 手动更新啊，不用等商店的版本

卸载商店的重新安装最新是可以，因为商店版本里设置很多，所以想能直接升级，就不用去重新配置了，现在正在摸索二进制包的直接更新，但是找不到商店版本的alist的安装目录，还没弄好。

**Reply by fen9wu** [reply]
没那么复杂，下载ipk，然后在istoreos里面系统—安装包下面直接上传安装就好了，相当于升级，所以之前的配置都是保留的。另外alist本身是包含备份和恢复功能的。为以防万一，升级前先在alist里面备份

**Comment by zhlx2835** [comment]
还是提示这个，目前有解决办法了吗

**Reply by anwen-anyi** [reply, marked as answer]
更新 v3.37.4 如果是Docker 请注意是否获取到的是最新版镜像(v3.37.4)

**Reply by Hk4Fun** [reply]
> 更新 v3.37.4 如果是Docker 请注意是否获取到的是最新版镜像(v3.37.4)

我升级到最新还是报这个错

**Reply by spaceflyx** [reply]
感谢大神，闪电更新3.37.4，升级后115恢复正常，上传下载均可以使用。

**Reply by hnzhaihui** [reply]
> 感谢大神，闪电更新3.37.4，升级后115恢复正常，上传下载均可以使用。

可惜我的istroeos是22.03 现在3.37.4只有22.02，不知道能升不

**Comment by hnzhaihui** [comment]
我去，我istroeos折腾升级alist3.73.4，结果搞崩溃了，死活启动不了了，正在绝望准备重新从头安装配置的时候，万年不动的istroeos商店竟然更新alist了，哈哈哈还是热乎的，才更新了几分钟。

**Comment by Askfk** [comment]
求一个arm64的v3.37.4版本的镜像，国内到处pull不下来

**Reply by zhsama** [reply]
前段时间不是移除GFW了吗？又封上了？
